hierarchies-move/sources/accreditation.move

```diff
@@ -69,6 +69,7 @@ public(package) fun is_property_allowed(
         let maybe_property = accreditation.properties.try_get(property_name);
 
         if (maybe_property.is_none()) {
+            idx_properties_to_attest = idx_properties_to_attest + 1;
```

Comment:
Wouldn't it be better the move  the incrementer from line 82 https://github.com/iotaledger/hierarchies/blob/fc6eea4d043ff6823710f9f22c289609934e0ec0/hierarchies-move/sources/accreditation.move#L82  to line 70 (before `is_none()`  condition), instead of doubling it?